xsd/siri_model/siri_facility-v2.0.xsd

```diff
@@ -721,37 +721,22 @@ Rail transport, Roads and road transport
    </xsd:sequence>
   </xsd:sequence>
  </xsd:group>
-<!-- ===Vehicle FACILITY====================================== -->
-<xsd:complexType name="TypeOfVehicleFuelStructure">
-	<xsd:annotation>
-		<xsd:documentation>Classification of Vehicle Energy type. See Transmodel TypeOfFuel</xsd:documentation>
-	</xsd:annotation>
-	<xsd:sequence>
-		<xsd:element name="FuelType" type="TypeOfVehicleFuelEnumeration">
-			<xsd:annotation>
-				<xsd:documentation>Values for Vehicle fuel type. +SIRI v2.1</xsd:documentation>
-			</xsd:annotation>
-		</xsd:element>
-		<xsd:element name="VehicleFuelTypeOther" type="TypeOfValueStructure" minOccurs="0">
-			<xsd:annotation>
-				<xsd:documentation>An arbitrary vehicle fuel classification. +SIRI v2.1</xsd:documentation>
-			</xsd:annotation>
-		</xsd:element>
-	</xsd:sequence>	
-</xsd:complexType>
-<xsd:simpleType name="TypeOfVehicleFuelEnumeration">
-		<xsd:annotation>
-			<xsd:documentation>Values for Vehicle fuel type. Use of "other" requires the additional open ended VehicleFuelTypeOther to be filled</xsd:documentation>
-		</xsd:annotation>
-		<xsd:restriction base="xsd:NMTOKEN">
-			<xsd:enumeration value="unknown"/>
-			<xsd:enumeration value="petrol"/>
-			<xsd:enumeration value="diesel"/>
-			<xsd:enumeration value="naturalGas"/>
-			<xsd:enumeration value="biodiesel"/>
-			<xsd:enumeration value="electricity"/>
-			<xsd:enumeration value="hydrogen"/>
-			<xsd:enumeration value="other"/>
-		</xsd:restriction>
-</xsd:simpleType>
+ <!-- ===Vehicle FACILITY====================================== -->
+ <xsd:complexType name="TypeOfVehicleFuelStructure">
+  <xsd:annotation>
+   <xsd:documentation>Classification of vehicle energy type. See Transmodel TypeOfFuel. +SIRI v2.1</xsd:documentation>
+  </xsd:annotation>
+  <xsd:sequence>
+   <xsd:element name="TypeOfFuel" type="TypeOfFuelEnumeration">
+    <xsd:annotation>
+     <xsd:documentation>Values for vehicle fuel type. Use of 'other' requires the additional open ended VehicleFuelTypeOther to be filled.</xsd:documentation>
```

Comment:
Please change "VehicleFuelTypeOther" in the annotation to "OtherTypeOfFuel".

Reply:
See commit [57a9e66](https://github.com/SIRI-CEN/SIRI/pull/58/commits/57a9e66b98e77a06a9a15405a27f81466c350dff)